.circleci/config.yml

```diff
@@ -15,10 +15,15 @@ jobs:
             pip install --upgrade pip
             pip install .[dev]
       - run:
-          name: 'pylint'
+          name: 'pylint tap'
           command: |
             source /usr/local/share/virtualenvs/tap-square/bin/activate
-            pylint tap_square -d 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring'
+            pylint tap_square -d 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,too-many-public-methods'
+      #- run:
+      #    name: 'pylint tests'
+      #    command: |
+      #      source /usr/local/share/virtualenvs/tap-square/bin/activate
+      #      pylint tests -d 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,duplicate-code,no-name-in-module'
```

Comment:
Do we want to pylint the tests? I feel like this should either be deleted or uncommented.

Reply:
I do intend to turn it on when I get all the pylint warnings squared away, but we can remove it if you feel strongly